wdio.conf.js

```diff
@@ -0,0 +1,51 @@
+require('chai').should();
+var Promise = require('pinkie-promise');
+var selenium = require('selenium-standalone');
+var seleniumServer;
+
+exports.config = {
+    specs: [
+        //TODO: use wildcard when all tests will be migrated
+        './tests/functional/core.js'
+    ],
+
+    capabilities: [{
+        browserName: 'firefox'
+    }],
+
+    // Level of logging verbosity: silent | verbose | command | data | result | error
+    logLevel: 'silent',
+
+    baseUrl: 'file://' + __dirname + '/tests/functional',
+
+    // Default timeout for all waitFor* commands.
+    waitforTimeout: 10000,
+
+    // Default request retries count
+    connectionRetryCount: 3,
+
+    framework: 'mocha',
+
+    reporters: ['dot', 'allure'],
+
+    mochaOpts: {
+        ui: 'bdd'
+    },
+
+    onPrepare: function (config, capabilities) {
+        //TODO: use selenium-standalone service when it will be available
```

Comment:
it should be done with https://github.com/webdriverio/wdio-selenium-standalone-service but it haven't implemented yet